Makefile.am

```diff
@@ -287,6 +287,7 @@ clean-local:
 man: $(dist_man_MANS)
 
 .rst:
+	mkdir -p $$(dirname $@)
 	rst2man.py $< > $@
```

Comment:
Whilst one is mucking around with this, it would be better to sau

     rst2man.py "<$" > "$@.tmp" && mv -f "$@.tmp" "$@"

to protect against signals partway through processing the manual source.